frontend/src/utils.js

```diff
@@ -104,9 +104,29 @@ function copyToClipboard(str) {
   return result;
 }
 
+const LOCALIZE_NUMBERS = !!(
+  typeof Intl === 'object' &&
+  Intl &&
+  typeof Intl.NumberFormat === 'function'
+);
+
+const UNITS = ['B', 'kB', 'MB', 'GB'];
+function bytes(num) {
+  const exponent = Math.min(Math.floor(Math.log10(num) / 3), UNITS.length - 1);
+  const n = Number(num / Math.pow(1000, exponent));
+  const nStr = LOCALIZE_NUMBERS
+    ? n.toLocaleString(navigator.language, {
+        minimumFractionDigits: 1,
+        maximumFractionDigits: 1
+      })
```

Comment:
> I wasn't sure if this meant that my file sizes would be in Hebrew (which is not an available language for Send), but the rest of the site would be Japanese (or Bulgarian, if Japanese wasn't an available language for Firefox Send site + l20n).

Ah, yes. If you pass `navigator.languages` it will be `["he", "ja", 'bg"]`. Since all browsers carry NumberFormat data for `he`, we'll format into `he`.

If you want to link the raw Intl API to use the same languages as were negotiated for l20n, you need to retrieve the result of language negotiation for l20n.

I'm not sure how this API currently works, but it should be something along the lines of `document.l10n.locales` which should be the list of locales that l20n negotiated down to.
Since `he` is not available, the list would end up being sth like `['ja', 'bg', 'en-US']` assuming both `ja` and `bg` are available, and `en-US` is the last fallback.

In that case, passing that list fo the Intl API will do the right thing for you.

I haven't been working with `l20n.js` package in a while, so CC'ing @stasm, who's maintaining it - he will hopefully be able to point out how to retrieve the negotiated locale list from l20n to pass to the Intl API.